packages/cli/commands/sandbox/delete.js

```diff
@@ -178,6 +198,9 @@ exports.builder = yargs => {
     describe: i18n(`${i18nKey}.options.account.describe`),
     type: 'string',
   });
+  yargs.option('force', {
+    type: 'boolean',
+  });
```

Comment:
For these scripting options, do we need a descriptor since its more of a hidden thing?